xml/ru/docs/contributing_changes.xml

```diff
@@ -34,46 +34,48 @@ hg clone http://hg.nginx.org/nginx
 принятому в nginx.
 В случае возникновения затруднений в оформлении
 изучите, как оформлен исходный код nginx, и следуйте этому стилю в своём коде.
-Изменения скорее будут приняты, если их стиль соответствует окружающему коду.
+Изменения скорее будут приняты, если их стиль соответствует окружающему
+коду.
 </para>
 
 <para>
-Зафиксируйте изменения, чтобы создать
-<link url="https://www.mercurial-scm.org/wiki/ChangeSet">набор изменений</link>
-в Mercurial.
+Зафиксируйте изменения в вашем ответвлении (fork) проекта, создав
+<link url="https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/about-commits">набор изменений</link> (commit).
```

Comment:
Github documentation has Russian translation (replace `en` with `ru` in links).

Reply:
thanks, links updated accordingly